src/phoenix/trace/semantic_conventions.py

```diff
@@ -61,6 +61,15 @@ def _missing_(cls, v: Any) -> Optional["MimeType"]:
         return None if v else cls.TEXT
 
 
+EMBEDDING_EMBEDDINGS = "embedding.embeddings"
```

Comment:
may be sufficient to call this just embeddings?
```suggestion
EMBEDDINGS = "embeddings"
```

Reply:
Where would the embedding model name and other metadata go in this case?

Reply:
they can go into the embedding object

Reply:
mostly because we're running out of words. we are using EMBEDDING in two places...

Reply:
Certainly possible, but that lead to some redundancies.

```
[
    {
        "embedding": [...],
        "text": "text-0",
        "model_name": "text-embedding-ada-002"
    },
    {
        "embedding": [...],
        "text": "text-1",
        "model_name": "text-embedding-ada-002"
    }
]
```